Automattic/delphin
# Refactor and simplify layouts

This pull request seeks to greatly simplify the set of layouts used to display pages.

The complexity of layouts has increased a lot lately, to the point that it's now very hard to reason about and to maintain them. However the recent redesign helped in deprecating a few custom rules, and putting aside some leftovers, simplified the set of layouts that are actually needed. This pull request thus consolidates all existing layouts into only two layouts: one with a light background (white) and another one with a dark background (light gray).
#### Testing instructions
1. Run `git checkout update/layouts` and start your server, or open a [live branch](https://delphin.live/?branch=update/layouts)
2. Open the [`Home` page](http://delphin.localhost:1337/)
3. Check that all pages display as before
#### Additional notes

This pull request also removes the fixed background hack introduced in https://github.com/Automattic/delphin/pull/709 thanks to the handy [`react-body-classname`](https://github.com/PactCoffee/react-body-classname) module.
#### Merging instructions

**This pull request is based on the branch from https://github.com/Automattic/delphin/pull/752**. As such - if I'm getting this right - it should be merged first (i.e. `update/layouts` into `add/search-page-footer`). Only then will it be possible to merge https://github.com/Automattic/delphin/pull/752 (i.e. `add/search-page-footer` into `master`).
#### Reviews

@Automattic/sdev-feed
